README.md

```diff
@@ -100,7 +100,7 @@ At printing time, these styles will:
   that they should:
 
   * ensure the table header (`<thead>`) is [printed on each page spanned by the
-    table](http://css-discuss.incutio.com/wiki/Printing_Tables)
+    table](https://www.smashingmagazine.com/2015/01/designing-for-print-with-css/#page-breaks)
```

Comment:
I don't think this link covers everything we have here, does it? I wonder if there's something that is closer.